hs/SvgParsing/Generator.hs

```diff
@@ -163,17 +167,26 @@ convertRectToSVG courseMap rect
                                                  (getArea (shapeId_ rect)))
                ! S.customAttribute "text-rendering" "geometricPrecision"
                ! S.customAttribute "shape-rendering" "geometricPrecision"
-               ! A.style (stringValue style) $
+               -- TODO: Remove the reliance on the colours here
+               ! (if class_ /= "hybrid"
+                  then
+                      A.style (stringValue style)
+                  else
+                      mempty)
+               $
                do S.rect ! A.rx "4"
                          ! A.ry "4"
                          ! A.x (stringValue . show . fst $ shapePos rect)
                          ! A.y (stringValue . show . snd $ shapePos rect)
                          ! A.width (stringValue . show $ shapeWidth rect)
                          ! A.height (stringValue . show $ shapeHeight rect)
-                         ! A.style (stringValue $ "fill:" ++
-                                    getFill (shapeId_ rect) ++
-                                    ";")
-                  concatSVG $ map (convertTextToSVG (shapeType_ rect))
+                         ! if class_ /= "hybrid"
```

Comment:
Why does the code not add styles for hybrids anymore?
![black_courseography](https://cloud.githubusercontent.com/assets/6301000/7151258/3e942e72-e2f5-11e4-8bb7-1ff3b03aeb50.png)